download.html

```diff
@@ -102,6 +102,11 @@
         Windows
     </h4>
     <p>
+		<p><a class='download-link' href='#installers'>Download</a>
+              the Gauge Windows Installer or you can install Gauge using Chocolatey</p>
```

Comment:
Could you please add a reference link to Chocolatey? Just in case someone hasn't heard of it.